qld_ref_nfs_playbook.yml

```diff
@@ -0,0 +1,18 @@
+- hosts: qld_ref_nfs
+  become: true
+  vars:
+  - nfs_exports:
+    - "/mnt/vdb/alphafold_db {{ hostvars['pulsar-qld-gpu1'].internal_ip.split('.')[:-1] | join('.') }}.0/24(rw,async,no_root_squash,no_subtree_check)"
```

Comment:
There is no `internal_ip` in this case. Shouldn't it be `ansible_ssh_host`?

Reply:
indeed! thank you